### DISCUSSION TITLE
KWH meter code update

### DISCUSSION THREAD

**Discussion by Delusional112** [huizebruin/s0tool#45] [General]
Hi,

I have a KWH meter connected to my electric car charger. It's an LEM022SJ with 400 pulses per KWH. I added the 400 to the options in the code.

I would like to read the KWH usage of the car charger. I have the ESP connected and online. But it is reading usage even if I'm not charging the car.

Danny

**Comment by huizebruin** [comment]
Hello Danny,

Dit u wire it good.
and why that low puls counter / kwh 
i will put 400 also in the next release

and did u put a resistor between the D5 and the 5v line about a 10kohm?

**Reply by Delusional112** [reply]
<img width="1014" alt="PNG-afbeelding" src="https://user-images.githubusercontent.com/79289457/194747591-441dd80d-488b-4d18-a4fb-229abdabaa57.png">
Hi,

I don't know why it is 400. The manual says it is. The electrician installed it with the car charger.

I didn't used a resistor. I will add it and wait for your next release. 

Thanks

**Reply by huizebruin** [reply]
Hello Danny 

I pushed a new version, and its now including LEM022SJ in the flash page.
[https://huizebruin.github.io/s0tool/](https://huizebruin.github.io/s0tool/)

**Reply by Delusional112** [reply]
Thanks, going to test it right away.

**Reply by huizebruin** [reply]
and how its working ?
